packages/admin-ui/src/components/smooth.scss

```diff
@@ -0,0 +1,46 @@
+.title {
```

Comment:
i would change this identifier to something more "smooth related" not that generic